internal/turso/organizations.go

```diff
@@ -116,8 +121,11 @@ func (c *OrganizationsClient) Usage() (OrgUsage, error) {
 	}
 	defer r.Body.Close()
 
```

Comment:
While you are at it, could you add a check here for 200 status code and return an error otherwise?